python/sbp/jit/parse_float.py

```diff
@@ -0,0 +1,49 @@
+import cffi
```

Comment:
Prior to merging this we need to figure out how to get this module to generate itself automatically (for development), then build a compiled version for a PyPI wheel (this is optional, since it might be worthy of a whole separate PR to figure out to build and distribute this).

Reply:
How about a first phase remedy:

```
from sbp.jit import parse_float
from sbp.jit import parse_float_c
```

Also note the additions to `tox.ini` so another possibility would be to add the requirement to run `make python` before usage but that would probably go unnoticed in quite a lot of cases.

Reply:
@pmiettinen that might work, as long as the import succeeds, hopefully this doesn't require the user to have write access to wherever `sbp.jit.parse_float` is installed?

Reply:
Pretty sure the write access is required but I don't think this it matters at this phase as this solution is aimed only for development and the files shall reside under the repository? Do this now and later a better solution in a PR adding the `sbp.jit` package into the release PyPi package?

Reply:
Ok